### ISSUE TITLE
 [userTypeFix] Code Coverage: Create tests for src/screens/UserPortal/Home/Home.tsx

### ISSUE THREAD

**Issue by varshith257** [PalisadoesFoundation/talawa-admin#1769]
We will need unittests done for all methods, classes and functions found in this file.

Any widgets, components, modals referenced in this file must also have unittests done

PR Acceptance Criteria

- When complete this file must show at least 100% coverage when merged into the code base.
- The PR will show a report for the code coverage for the file you have added. You can use that as a guide

**Comment by varshith257** [comment]
@Shubh152 Get it assigned and submit a PR once it is done

**Comment by Shubh152** [comment]
please assign this to me.

**Comment by varshith257** [comment]
@Shubh152 Complete it and get it assigned next file issue

**Comment by palisadoes** [comment]
We are tracking this issue with one of our projects, and we'd like to track its progress especially as the GSoC announcement dates draw near.

1. When you create your PR, please link it back to this project:
    1. https://github.com/orgs/PalisadoesFoundation/projects/24/views/1
1. The process is simple. It can be found here:
    1. https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#manually-linking-a-pull-request-to-an-issue-using-the-pull-request-sidebar

**Comment by GlenDsza** [comment]
@palisadoes @Shubh152 For PR #1826 had to fix & refactor `src/screens/UserPortal/Home/Home.tsx` so added test case for it as well once PR is merged Test Coverage would be 100% for this file.

**Comment by palisadoes** [comment]
Fixed in this PR
- https://github.com/PalisadoesFoundation/talawa-admin/pull/1826
